packages/image_picker/image_picker_linux/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.2.2
```

Comment:
A bugfix for a 0.x package uses the `+` field, so this should be `0.2.1+2`.

Reply:
Fixed.